src-docs/src/views/combo_box/tool_tips.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+
+import { EuiComboBox } from '../../../../src/components';
+import { DisplayToggles } from '../form_controls/display_toggles';
+
+const optionsStatic = [
+  {
+    label: 'Titan',
+    'data-test-subj': 'titanOption',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Enceladus is disabled',
+    disabled: true,
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Mimas',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Dione',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Iapetus',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Phoebe',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Rhea',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label:
+      "Pandora is one of Saturn's moons, named for a Titaness of Greek mythology",
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Tethys',
+    toolTipContent: 'Lorem ipsum',
+  },
+  {
+    label: 'Hyperion',
+    toolTipContent: 'Lorem ipsum',
+  },
```

Comment:
```suggestion
  {
    label: 'Titan',
    toolTipContent: 'Titan is the largest moon of Saturn and the second-largest in the Solar System',
  },
  {
    label: 'Pandora',
    toolTipContent: 'Pandora is one of Saturn's moons, named for a Titaness of Greek mythology',
  },
  {
    label: 'Iapetus',
    toolTipContent: 'Iapetus is the outermost of Saturn's large moons',
    toolTipProps: { position: 'bottom' },
  },
```

Let's provide fewer options here in favor of more specific examples of the kind of tooltip content that would be useful to use the prop for!